src/ResourceManager/Network/Commands.Network/help/New-AzureRmPublicIpAddress.md

```diff
@@ -47,6 +48,18 @@ pointing to the $customFqdn specified in the command. As a pre-requisite, the $c
 webapp.contoso.com) should have a DNS CNAME record (forward-lookup) pointing to
 $dnsPrefix.$location.cloudapp.azure.com.
 
+### 3: Create a new public IP address with IpTag
+```
+$publicIp = New-AzureRmPublicIpAddress -Name $publicIpName -ResourceGroupName $rgName -AllocationMethod Static -DomainNameLabel $dnsPrefix -Location $location -IpTags ipTag
```

Comment:
Add the New-AzureRmPublicIpTag cmdlet to this example so people know how to create the object.

Reply:
done

Reply:
Please also add it to the content of the example itself:

```
$ipTag = New-AzureRmPublicIpTag -IpTagType "FirstPartyUsage" -Tag "/Sql"
$publicIp = New-AzureRmPublicIpAddress -Name $publicIpName -ResourceGroupName $rgName -AllocationMethod Static -DomainNameLabel $dnsPrefix -Location $location -IpTags ipTag
```

Reply:
done